### ISSUE TITLE
Expandable sections not robust, no unobtrusiveness

### ISSUE THREAD

**Issue by robustusability** [w3c/wai-wcag-quickref#138]
The new design of the quickref looks very (!) nice in many ways, but it is quite self-contradicting that a page about following WCAG fails to meet one of the basic requirements - robustness. If JavaScript does not work or fails to load correctly, the expandable sections are not shown (i.e. "Show techniques..."). The old page worked in basically any browser or device.

Please check out the suggestions at: https://www.w3.org/wiki/The_principles_of_unobtrusive_JavaScript

**Comment by yatil** [comment]
JavaScript is accessibility supported, the content is available through other means as well (un-expanded it is basically an overview to WCAG itself). We want to avoid unnecessary jumping around of sections when JavaScript is available – which would happen when you show the sections and then hide them on load. If you have good suggestions to avoid that, I’m happy to look into it eventually.

**Comment by robustusability** [comment]
If hiding on load is seen as a problem (it can usually be done very fast), the issue could be solved with AJAX, so that with JavaScript the asked-for content is loaded and inserted into the page, while a user agent not supporting JavaScript get a url to a new page, displaying only the same content.
Reloading is, at least, better than not being able to see the content.
